app/services/ect_at_school_periods/search.rb

```diff
@@ -13,6 +13,17 @@ def ect_periods(urn: nil, trn: nil)
       @scope
     end
 
+    def current_school(trn:)
+      teacher = Teacher.find_by(trn:)
+      return unless teacher
```

Comment:
Totally doesn't matter but this is quite concise on one line.

```suggestion
      return unless (teacher = Teacher.find_by(trn:))
```

Reply:
Ah yes that is nicer but I have removed this whole method now.

See my comment here: https://github.com/DFE-Digital/register-early-career-teachers-public/pull/554#discussion_r2098483942